Specifications/Embeddings.Processing/for_CompareProjectionStates/when_comparing/and_the_states_are_equal.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Dolittle.Runtime.Projections.Store.State;
+using Dolittle.Runtime.Rudimentary;
+using Machine.Specifications;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Dolittle.Runtime.Embeddings.Processing.for_CompareProjectionStates.when_comparing
+{
+    public class and_the_states_are_equal
+    {
+        static CompareProjectionStates comparer;
+        static ProjectionState left;
+        static ProjectionState right;
+
+        Establish context = () =>
+        {
+            dynamic left_dynamic = new JObject();
+            left_dynamic.FirstProp = "FirstProp";
+            left_dynamic.Dictionary = new JObject
+            {
+                { "first_key", "first_value" },
+                { "second_key", "second_value" },
+                { "third_key", "third_value" }
+            };
+            left_dynamic.Array = new JArray
+            {
+                "First",
+                "Second",
+                "Third"
+            };
+            left = new ProjectionState(JsonConvert.SerializeObject(left_dynamic));
+
+            dynamic right_dynamic = new JObject();
+            right_dynamic.FirstProp = "FirstProp";
+            right_dynamic.Dictionary = new JObject
+            {
+                { "first_key", "first_value" },
+                { "second_key", "second_value" },
+                { "third_key", "third_value" }
+            };
+            right_dynamic.Array = new JArray
+            {
+                "First",
+                "Second",
+                "Third"
+            };
```

Comment:
To make the establishing more error prone I would make a method call SetState(JObject) that sets the wanted props on the object instead of copy-pasta

Reply:
You mean: to create a method `SetState(JObject objectToSet)` in the spec, that sets these properties, so that it becomes _less_ error prone, right?

Reply:
Yes